src/libcore/time.rs

```diff
@@ -204,7 +204,7 @@ impl Duration {
     /// [`subsec_nanos`]: #method.subsec_nanos
     #[stable(feature = "duration", since = "1.3.0")]
     #[inline]
-    pub fn as_secs(&self) -> u64 { self.secs }
+    pub const fn as_secs(&self) -> u64 { self.secs }
```

Comment:
can you add `#[rustc_const_unstable="duration_getters"]` to the newly const-ed functions so they aren't immediately constant on the next stable rust?